Print --help when no options are entered.
This fixes annoying bug/feature that the script crashes when no file attribute is provided
diff -ru officeparser-fce88741d3f78d5588bb214b824c2d51232287a5/officeparser.py officeparser-fce88741d3f78d5588bb214b824c2d51232287a5.new/officeparser.py
--- officeparser-fce88741d3f78d5588bb214b824c2d51232287a5/officeparser.py	2023-11-15 11:13:50.897816166 +0100
+++ officeparser-fce88741d3f78d5588bb214b824c2d51232287a5.new/officeparser.py	2023-11-15 11:14:00.324157515 +0100
@@ -461,6 +461,9 @@
 
 def _main():
 
+    if len(sys.argv) < 2:
+        sys.argv.append('--help')
+
     parser = OptionParser()
 
     parser.add_option('-l', '--log-level', dest='log_level',
